app/hooks/useConfirmDialog.tsx

```diff
@@ -6,10 +6,10 @@ import ReactDOM from 'react-dom';
 import snackbar from '@/services/SnackbarUtils';
 
 const textDict = {
-  probe: 'Do you wish to remove Ready and Reviewer signatures?',
-  pharmacogenomic: 'Do you wish to remove Ready and Reviewer signatures?',
-  genomic: 'Do you wish to remove Author and Reviewer signatures?',
-  rapid: 'Do you wish to remove Author and Reviewer signatures?',
+  probe: 'Making this change will cause signatures to be removed. do you want to proceed?',
+  pharmacogenomic: 'Making this change will cause signatures to be removed. do you want to proceed?',
+  genomic: 'Making this change will cause the reviewer and analyst signatures to be removed. do you want to proceed?',
+  rapid: 'Making this change will cause the reviewer and analyst signatures to be removed. do you want to proceed?',
```

Comment:
capitalize 'Do' for each

Reply:
🤦  oh dear copy and paste

Reply:
[ea69abc](https://github.com/bcgsc/pori_ipr_client/pull/402/commits/ea69abc2c2f90546ead1199e7da4e1a321ef5f95)